kitsune/users/models.py

```diff
@@ -72,14 +73,14 @@ class Profile(ModelBase, SearchMixin):
         validators=[TwitterValidator],
         verbose_name=_lazy("Twitter Username"),
     )
-    facebook = models.URLField(
-        max_length=255, null=True, blank=True, verbose_name=_lazy("Facebook URL")
+    community_mozilla_org = models.CharField(
+        max_length=255, null=True, blank=True, verbose_name=_lazy("Mozilla Community Username")
     )
-    mozillians = models.CharField(
-        max_length=255, null=True, blank=True, verbose_name=_lazy("Mozillians Username")
+    people_mozilla_org = models.CharField(
+        max_length=255, null=True, blank=True, verbose_name=_lazy("Mozilla Directory Username")
     )
-    irc_handle = models.CharField(
-        max_length=255, null=True, blank=True, verbose_name=_lazy("IRC nickname")
+    matrix_handle = models.CharField(
+        max_length=255, null=True, blank=True, verbose_name=_lazy("Matrix nickname")
```

Comment:
`Matrix username` isn't capitalised like the above verbose names, should we make it `Matrix Nickname`?